adagrams/game.py

```diff
@@ -1,11 +1,150 @@
+import random
 def draw_letters():
-    pass
+   letter_pool = {
+    'A': 9,
+    'B': 2,
+    'C': 2,
+    'D': 4,
+    'E': 12,
+    'F': 2,
+    'G': 3,
+    'H': 2,
+    'I': 9,
+    'J': 1,
+    'K': 1,
+    'L': 4,
+    'M': 2,
+    'N': 6,
+    'O': 8,
+    'P': 2,
+    'Q': 1,
+    'R': 6,
+    'S': 4,
+    'T': 6,
+    'U': 4,
+    'V': 2,
+    'W': 2,
+    'X': 1,
+    'Y': 2,
+    'Z': 1
+     }
+   #taking an empty list to store drawn letters
+   drawn_letters =[]
```

Comment:
Missing white space after equal sign, should look like `drawn_letters = []` 

It's a small thing, but as you write more code (and eventually contribute to a large codebase) it's important to be consistent with whitespaces so that the whole project stays neat. Here's the official Python [style guide](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements) that covers whitespaces.